packages/terra-functional-testing/src/services/wdio-selenium-docker-service.js

```diff
@@ -66,6 +66,12 @@ class SeleniumDockerService {
 
     await exec(`TERRA_SELENIUM_DOCKER_VERSION=${this.version} docker stack deploy -c ${composeFilePath} wdio`);
 
+    // There are occasions where the docker stack deployment is not completely ready when running back-to-back wdio test runs.
+    // As a result, the test run is unable to start and receives the 'Error forwarding the new session cannot find : Capabilities {browserName: chrome}' error.
+    // This waits for 5 seconds and ensure the services and network are created and ready.
+    await this.wait(5000);
```

Comment:
:(

Reply:
Is it possible we need to "beef" up the removal of the network and service? RemoveStack only wait for the tear down of the wdio service/network if the stack is found. I think it's robust enough it could always be called:

https://github.com/cerner/terra-toolkit/blob/128260b396da3fda433e552a52724de35669f99f/packages/terra-functional-testing/src/services/wdio-selenium-docker-service.js#L89

Reply:
One additional thing we could check as part of the stack removal is to ensure we can no longer connect to http://localhost:4444/wd/hub/status. Kind of similar to [waitForNetworkReady](https://github.com/cerner/terra-toolkit/blob/128260b396da3fda433e552a52724de35669f99f/packages/terra-functional-testing/src/services/wdio-selenium-docker-service.js#L203) but resolve if we can no longer connect to it. However, I don't think doing so adds anything to fix this error.

Reply:
Is this wait is only required when running wdio multiple times sequentially. I wonder if we're punishing everyone for a case most common with ourselves. Should we instead inject the wait in the script?

Reply:
Wouldn't injecting the wait in the wdio script would only fix it for us but not for consumers? They would also need to add the wait themselves in their script.

Reply:
I think @mjhenkes meant the travis script in the .travis.yml file